.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+test/test-bundle.js
```

Comment:
Should this be a separate pr?

Reply:
just including it because it was missed in the last PR when allowing cli testing